File: api-js/src/notify/notify-client.js
const { NotifyClient } = require('notifications-node-client')

const { NOTIFICATION_API_KEY, NOTIFICATION_API_URL } = process.env

const notifyClient = new NotifyClient(
  NOTIFICATION_API_URL,
  NOTIFICATION_API_KEY,
)

module.exports = {
  notifyClient,
}

File: scanners/dns-processor/notify/notify_client.py
import os
from notifications_python_client.notifications import NotificationsAPIClient
from dotenv import load_dotenv

load_dotenv()
notify_client = NotificationsAPIClient(
    api_key=os.getenv("NOTIFICATION_API_KEY"),
    base_url=os.getenv("NOTIFICATION_API_URL"),
)
